docs/self-managed/operational-guides/backup-restore/modeler-backup-and-restore.md

```diff
@@ -0,0 +1,41 @@
+---
+id: modeler-backup-and-restore
+title: Backup and restore Web Modeler data
+description: "How to perform a backup and restore of Web Modeler data."
+keywords: ["backup", "backups"]
+---
+
+## Create backup
+
+To create a backup of Web Modeler data, you must back up the database that Web Modeler uses by following the instructions of the official [PostgreSQL documentation](https://www.postgresql.org/docs/current/backup-dump.html).
+
+For example, to create a backup of the database using `pg_dumpall`, use the following command:
+
+```bash
+pg_dumpall -U <DATABASE_USER> -h <DATABASE_HOST> -p <DATABASE_PORT> | tee dump.psql
+Password: <POSTGRES_POSTGRES_PASSWORD>
+```
+
+`pg_dumpall` may ask multiple times for the same password.
+The database will be dumped into `dump.psql`.
+
+## Restore
+
+Backups can only be restored with downtime.
+To restore the database dump, first ensure that Web Modeler is stopped.
+Then, to restore the database use the following command:
+
+```bash
+psql -U <DATABASE_USER> -h <DATABASE_HOST> -p <DATABASE_PORT> -f dump.psql
+```
+
+After the database has been restored, you can start Web Modeler again.
+
+:::warning
+When restoring Web Modeler data from a backup, ensure that the ids of the users stored in your OIDC provider (e.g. Keycloak) do not change in between the backup and restore.
```

Comment:
FYI: This will be documented more thoroughly with https://github.com/camunda/web-modeler/issues/7583